src/Core/Resolvers/Factories/QueryManagerFactory.cs

```diff
@@ -49,23 +67,23 @@ public QueryManagerFactory(RuntimeConfigProvider runtimeConfigProvider, ILogger<
                         break;
                     case DatabaseType.MSSQL:
                         queryBuilder = new MsSqlQueryBuilder();
-                        exceptionParser = new MsSqlDbExceptionParser(runtimeConfigProvider);
-                        queryExecutor = new MsSqlQueryExecutor(runtimeConfigProvider, exceptionParser, logger, contextAccessor);
+                        exceptionParser = new MsSqlDbExceptionParser(_runtimeConfigProvider);
+                        queryExecutor = new MsSqlQueryExecutor(_runtimeConfigProvider, exceptionParser, _logger, _contextAccessor, _handler);
```

Comment:
Added the comment to the PR description. It is there in some classes because when they configure there state (such as during a hot-reload) they have classes they create which will need to be registering hot-reload events. The ones that don't save a handler dont need worry about this case.